src/components/Button.js

```diff
@@ -22,10 +25,19 @@ const Button = styled.button`
   }
 `;
 
-export const ListButton = ({ title, onClick }) => {
+export const ListButton = ({ title, audiofile }) => {
+  const [playing, setPlaying] = useState(false);
+
+  const toggle = () => setPlaying(!playing);
+
+  useEffect(() => {
+    const audioElement = new Audio(audiofile);
+    playing ? audioElement.play() : audioElement.pause();
+  }, [playing, audiofile]);
```

Comment:
```suggestion
 const audioElement = useRef(new Audio(audiofile));

  const toggle = () => {
    playing ? audioElement.current.pause() : audioElement.current.play();
    setPlaying(!playing);
  };
```

this one is tested and should work :) 

- your ternary was in the wrong order -> if it is playing, pause .. if its not playing play 🦸  
- last time we dealt with audio was with vanilla javascript. In React you have to use `useRef()`, otherwise react won't know after render, wich element you wanna stop [Resource](https://medium.com/javascript-in-plain-english/implementing-useref-in-react-732908aa1998)
-  you don't need a useEffect() 🚜
- your toggle function got the whole power

Happy listening :)

Reply:
THX for your comment! To add useRef was the missing hint. But I still have to use the useEffect. If not, I got the error "too many re-renders...".
But now it works ;-) 😃

Reply:
ah really? interesting 🙃 
hope you'll make some people happy with your project!